### ISSUE TITLE
Java Memory Assistant: deprecated and wrong link

### ISSUE THREAD

**Issue by calle2010** [SAP-docs/btp-cloud-platform#192]
### Issue description

https://help.sap.com/docs/btp/sap-business-technology-platform/java-memory-assistant-aws-azure-or-gcp-regions?q=volume

The link 
[Java Memory Assistant](https://help.sap.com/docs/link-disclaimer?site=https%3A%2F%2Fgithub.com%2FSAP%2Fjava-memory-assistant)
leads to a GitHub repository which has a deprecation notice sind September 1st.

The link
[Java Memory Assistant Framework](https://help.sap.com/docs/link-disclaimer?site=https%3A%2F%2Fgithub.com%2Fcloudfoundry%2Fjava-buildpack%2Fblob%2Fmaster%2Fdocs%2Fframework-java_memory_assistant.md)
leads to branch "master", which doesn't exist anymore, and should be updated to "main".

Overall it is unclear if the Java Memory Assisstant should still be used and if there is a replacement for the deprecated component.

### Feedback Type (Optional)

outdated information

### Page Title on SAP Help Portal (prefilled)

Java Memory Assistant [AWS, Azure, or GCP Regions]

### Page URL on SAP Help Portal (prefilled)

https://help.sap.com/docs/btp/sap-business-technology-platform/java-memory-assistant-aws-azure-or-gcp-regions?q=volume

**Comment by annawenger20** [comment]
Hi @calle2010 - thanks a lot for this feedback as well. We'll look into it and get back to you if there are further questions.

**Comment by Joysie** [comment]
Hi @calle2010,

Thank you for your valuable feedback and contribution! 

I corrected the second link. 

About the deprecated repository - you're right. Nevertheless, the component is still working fine and SAP Java Buildpack is using it ([version 0.5.0](https://github.com/SAP-archive/java-memory-assistant/releases/tag/0.5.0)). For the time being, there is no replacement of this component. 

I updated the documentation, adding also a relevant disclaimer. The changes will be available on SAP Help Portal with the next SAP BTP update.

Best regards,
Gergana
